runner_manager/models/runner.py

```diff
@@ -25,7 +26,6 @@
 
 class RunnerStatus(str, Enum):
     online = "online"
-    idle = "idle"
     offline = "offline"
```

Comment:
For the record I double checked your statement regarding status @Abubakarr99, and indeed discovered that there's no such thing as a `idle` status, only online and offline, then a runner is considered as online or active with the `busy` flag. Made the according changes.